project-generation/content/templates/ci/scripts/build.tmpl

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+export GOPATH=$cwd/go
+
+pushd $GOPATH/src/github.com/ONSdigital/{{.Name}}
+  make build && cp build/{{.Name}} $cwd/build
+  cp Dockerfile.concourse $cwd/build
```

Comment:
Copying in a `Dockerfile.concouse` but this isn't included in the template repo

Reply:
Not everything in the template directory is currently used  - For this task everything at the root directory of templates is used and everything in the .github directory. The other files came from the POC. As this task is just the generic content, not the API content those templates are just copied from the POC and will be worked into the project on the next templating task